lib/tools.js

```diff
@@ -0,0 +1,94 @@
+const path = require('path');
```

Comment:
As a being downloaded tools, giving permission is required.
(As matching each os platforms)

Reply:
maybe after unzip, the permission should be granted.

Reply:
done